pkgs/development/interpreters/python/hooks/default.nix

```diff
@@ -230,6 +230,29 @@ in
     } ./python-recompile-bytecode-hook.sh
   ) { };
 
+  pythonRelaxBuildDepsHook = callPackage (
+    {
+      makePythonHook,
+      packaging,
+      tomlkit,
+    }:
+    makePythonHook
+      {
+        name = "python-relax-build-deps-hook";
+      }
+      (writeTextFile {
+        name = "python-relax-build-deps-hook.sh";
+        text = ''
+          _pythonRelaxBuildDepsHook() {
+            PYTHONPATH="${packaging}/${pythonSitePackages}:${tomlkit}/${pythonSitePackages}:$PYTHONPATH" \
+              ${pythonInterpreter} ${./python-relax-build-deps-hook.py}
+          }
+
+          preConfigureHooks+=(_pythonRelaxBuildDepsHook)
+        '';
+      })
```

Comment:
This could be plonked in a text file (using `makeHook`'s substitution mechanism) but it seems worse:
anyone reading the code would have to first open `python-relax-build-deps-hook.sh`, only to find out it's a trivial stub and follow to the actual implementation instead.